aten/src/ATen/native/quantized/cpu/conv_serialization.h

```diff
@@ -167,9 +152,6 @@ ConvParamsSerializationType serialize_conv(
   auto output_padding = params->output_padding().vec();
   params_vec.insert(params_vec.end(), output_padding.begin(),
                     output_padding.end());
-  for (int i = 0; i < kSpatialDim; i++) {
```

Comment:
Seems like this was left in https://github.com/pytorch/pytorch/pull/39714 by accident, and tests did not catch it.  This PR adds test coverage for this path, so we need to fix it to land.